powerform/pf-deployment-context/functions/private/deploy-pfdefinitions.ps1

```diff
@@ -8,8 +8,7 @@ Function Deploy-PfDefinitions
 
     if($PfDefinitions){
         Foreach($definition in $PfDefinitions){
-            $access = (New-Object -TypeName "$($definition.ClassPrefix)Access" -ArgumentList $definition)
-            $state = $access.GetOrSet()
+            $state = $definition.Deploy()
```

Comment:
Mentioned this above.  This is the only way that came to mind to quickly deploy resources.  The Deploy method on the definition calls a resource specific function for the deploy.